docs/resources/User.md

```diff
@@ -67,7 +67,7 @@ There are other rules and restrictions not shared here for the sake of spam and
 | 0       | None                     | None                                                                                                                                           |
 | 1 << 0  | STAFF                    | Discord Employee                                                                                                                               |
 | 1 << 1  | PARTNER                  | Partnered Server Owner                                                                                                                         |
-| 1 << 2  | HYPESQUAD                | HypeSquad Events Coordinator                                                                                                                   |
+| 1 << 2  | HYPESQUAD                | HypeSquad Events                                                                                                                               |
```

Comment:
I believe it sounds less empty, what do you think?
```suggestion
| 1 << 2  | HYPESQUAD                | HypeSquad Events Member                                                                                                                        |
```

Reply:
the client doesn't use the word "member" so i don't it should be used here

Reply:
I would tend to agree with Rodry that "Member" is a good replacement to align with the other descriptions in the table. This table does not necessarily reflect the names shown in the client.